docs/user-guide/administration/manage-etcd.md

```diff
@@ -245,3 +300,178 @@ spec:
 ```
 
 Restart GreptimeDB Metasrv to complete etcd restore.
+
+## Monitoring
+
+- Prometheus Operator installed (e.g. via [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)).
+- podmonitor CRD installed (automatically installed with Prometheus Operator).
+
+Add the following to your `etcd-monitoring.yaml` to enable monitoring:
+
+```yaml
+replicaCount: 3
+
+auth:
+  rbac:
+    create: false
+  token:
+    enabled: false
+
+persistence:
+  storageClass: null
+  size: 8Gi
+
+resources:
```

Comment:
Based on https://etcd.io/docs/v3.5/op-guide/hardware/, I think the default setting will be better to use `2C/8G`.

Reply:
Fixed